src/content/getting-started/prepare-your-provider-account/azure/_index.md

```diff
@@ -3,11 +3,169 @@ title: Prepare your provider account for Azure
+
+    * An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
+    * [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed and configured to point to chosen subscription via [az account set](https://learn.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-set) command.
+
+### Service quotas {#quotas}
+
+Azure enforces [service quota through all the cloud services](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). The following overview lists the usual resources you may adjust depending on number of clusters and their size planned to be deployed:
+
+![Azure quotas list](./quotas_general.png)
+
+Access to the [Quotas portal service](https://portal.azure.com/#view/Microsoft_Azure_Capacity/QuotaMenuBlade/~/myQuotas) to check and adjust quotas as required. You can also use the Azure CLI to check quotas like:
+
+```sh
+az vm list-usage --location <region>
+```
+
+If your current quotas are insufficient, you can request an increase. In the portal, there is no CLI command, follow the steps below:
+
+* Go to the **Quotas** section in the [Azure portal](https://portal.azure.com/#view/Microsoft_Azure_Capacity/QuotaMenuBlade/~/myQuotas).
+* Click on pencil button to **request an update**.
+* Fill the new limit and submit the request.
+
+<img src="./quota_editing.png" style="border: solid 1px aliceblue; margin: 8px 0;" alt="Azure quotas edit" />
+
+You can follow the request status in the **Quotas** section.
+
+### Permissions
+
+There are two permissions roles that need to be created in the Azure subscription. One for the Giant Swarm controller used by the CAPZ controller in the management cluster to provision all infrastructure to manage workload clusters, and the other for Giant Swarm engineers to access the Azure account for support purposes.
+
+### Staff permissions {#iam-staff-role}
+
+Firstly, you need to grant access to Giant Swarm Ops/Support to your Azure subscription. Access to the portal is important for our every day support, where investigation and manual interventions are sometimes necessary.
+
+The easiest way is to create an [Azure Deployment Environment](https://azure.microsoft.com/en-us/products/deployment-environments) to delegate resource management to third parties. In this case, you must allow the Giant Swarm Staff group to manage your resources. This is beneficial as you don't have to manage access for each person separately within your subscription, instead you add a managed group that's kept up to date by Giant Swarm. There is a solution available called [Azure Lighthouse](https://learn.microsoft.com/en-us/azure/lighthouse/overview), which allows to delegate resource managements towards service providers such as Giant Swarm.
```

Comment:
Assuming that "Giant Swarm Staff" is the name of the group, please format it as a literal.

```suggestion
The easiest way is to create an [Azure Deployment Environment](https://azure.microsoft.com/en-us/products/deployment-environments) to delegate resource management to third parties. In this case, you must allow the `Giant Swarm Staff` group to manage your resources. This is beneficial as you don't have to manage access for each person separately within your subscription, instead you add a managed group that's kept up to date by Giant Swarm. There is a solution available called [Azure Lighthouse](https://learn.microsoft.com/en-us/azure/lighthouse/overview), which allows to delegate resource managements towards service providers such as Giant Swarm.
```